Dockerfile

```diff
@@ -18,9 +18,9 @@ EXPOSE 3000
 
 # Run the application
 CMD echo "Running Prisma commands..." && \
-    npx prisma generate && \
-    npx prisma db push && \
-    npx @snaplet/seed sync && \
-    npx prisma db seed && \
+    # npx prisma generate && \
```

Comment:
Revert. Not sure if this is a problem with this branch for you as well? If it is, I still think you can just run the commands separately on your machine